openapi.yaml

```diff
@@ -588,6 +579,16 @@ components:
       required: true
       schema:
         type: string
+    imageWidth:
+      in: query
+      name: filter[width]
```

Comment:
Hmm, not sure, but I think this parameter is not a "filter". I'm fine naming this for the conssitency, just curious your thoughts.

Reply:
I checked out Json API specification and only filtering parameters get to be filter[] params. I set it back to just `width`.